common/djangoapps/coursetalk/admin.py

```diff
@@ -0,0 +1,7 @@
+"""Manage coursetalk configuration. """
+from django.contrib import admin
+from config_models.admin import ConfigurationModelAdmin
+from coursetalk.models import CourseTalkWidgetConfiguration
```

Comment:
Small nitpick: we tend to group imports by where they come from: first is standard library, second is pip-installed modules, and modules from edx-platform last. It's a tiny thing (and lots of old code doesn't conform), but I like to make sure new code is an example of best practices.